api/app/Providers/AuthServiceProvider.php

```diff
@@ -34,6 +45,40 @@ public function boot()
             if ($request->input('api_token')) {
                 return User::where('api_token', $request->input('api_token'))->first();
             }
+            if ($request->bearerToken()) {
+                $bearerToken = $request->bearerToken(); // 1. extract JWT access token from request.
+                if ($bearerToken) {
```

Comment:
Since $bearerToken is equal to $request->bearerToken(), the second if statement is superfluous